scripts.html

```diff
@@ -4,16 +4,219 @@
 menu-id: l1-workflow2
 pdfs:
     - scripts.pdf
-description: Scripts allow users to add server-side functionality to the OMERO clients.
+description: Scripts allow additional server-side functionality to be added by users to the OMERO clients.
 version:
     - yes
```

Comment:
Can we add a redirect from the removed utility scripts page by adding:

```
redirect_from:
  - /utility-scripts.html
```

to the header?